cmd/dcrdata/internal/api/insight/converter.go

```diff
@@ -157,10 +158,19 @@ func (iapi *InsightApi) DcrToInsightBlock(inBlocks []*chainjson.GetBlockVerboseR
 	if dcp0010Height == -1 {
 		dcp0010Height = math.MaxInt64
 	}
+	dcp0012Height := iapi.BlockData.DCP0012ActivationHeight()
+	if dcp0012Height == -1 {
+		dcp0012Height = math.MaxInt64
+	}
 
 	RewardAtBlock := func(blocknum int64, voters uint16) float64 {
-		useDCP0010 := blocknum >= dcp0010Height
-		work, stake, tax := txhelpers.RewardsAtBlock(blocknum, voters, iapi.params, useDCP0010)
+		var ssv standalone.SubsidySplitVariant = standalone.SSVOriginal
```

Comment:
Not a big deal as it is correct regardless, but if you care about consistency, this declaration is inconsistent with the ones in `apiroutes.go` and `pgblockchain.go`.

```suggestion
		ssv := standalone.SSVOriginal
```